compiler-rt/lib/rtsan/rtsan.cpp

```diff
@@ -28,6 +29,13 @@ static void SetInitialized() {
   atomic_store(&rtsan_initialized, 1, memory_order_release);
 }
 
+static auto PrintDiagnosticsAndDieAction(DiagnosticsInfo info) {
```

Comment:
Could be const ref?

Reply:
We discussed this in person, we prefer for now (while the data is small and on the stack) to avoid complexities from potentially dangling references. We'll stick with value semantics and if lack of copy elision becomes a problem, change it later.